ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/RewardsActionsComposable.kt

```diff
@@ -74,38 +76,33 @@ fun ActionCard(
 ) {
   Card(
     modifier = Modifier
-      .padding(
-        start = 8.dp,
-        top = 16.dp
-      )
-      .width(166.dp)
-      .height(200.dp)
+      .size(width = 160.dp, height = 208.dp)
```

Comment:
Did you make the Action Card smaller? Because in future PR i will add the "Challenge Reward" card and i struggled to put the title in one line. PS: I already have it in titleSmall typography

Reply:
yes, on layout it is even smaller, but I changed the width because 166 is not multiple of 8, and it is too much bigger than xd.

you're right regarding titlesmall, I alrealdy changed it

https://xd.adobe.com/view/d5d0e051-89c6-4c75-930c-772e1a178286-8083/screen/166ca3c0-d996-4adc-aab9-87fbf2df63f2/specs/